.github/workflows/regression_test.yml

```diff
@@ -71,6 +71,12 @@ jobs:
             gpu-arch-type: "cuda"
             gpu-arch-version: "12.1"
 
+          - name: ROCM Nightly
+            runs-on: linux.rocm.gpu
+            torch-spec: '--pre torch --index-url https://download.pytorch.org/whl/nightly/rocm6.2.4'
+            gpu-arch-type: "rocm"
+            gpu-arch-version: "6.2.4"
+
```

Comment:
Move this section to the `test-nightly` job since that's where we are testing the nightly wheels, and it uses `linux_job_v2.yml`, which should be updated as per https://github.com/pytorch/test-infra/pull/6003#discussion_r1893202031